gui/src/app/mod.rs

```diff
@@ -119,6 +119,18 @@ impl App {
                 info!("Internal daemon stopped");
             }
         }
+        if let Some(exe_config) = &self.config.internal_bitcoind_exe_config {
+            let args = vec![
+                format!("-chain={}", &self.cache.network.to_core_arg()),
+                format!("-datadir={}", exe_config.data_dir.to_string_lossy()),
+                "stop".to_string(),
+            ];
+            info!("Stopping internal bitcoind");
+            let _res = std::process::Command::new("bitcoin-cli")
```

Comment:
I'm using `bitcoin-cli` here, but perhaps I should determine `bitcoin-cli`'s path based on `bitcoind`'s path, which is saved in the config file, or otherwise kill the `bitcoind` process directly.

Reply:
> Better yet, since you start the process in a thread i presume you have some sort of process handle? Can't you just stop the started process from this?

I've looked into this and sending a SIGTERM isn't portable to Windows, and other portable ways of doing this require new dependencies. Simply calling `stop` on `BitcoinD` make more sense.

Reply:
OK great thanks. My initial impression from a quick check was that keeping track of the child process would require bigger changes so I was tending towards the `stop()` approach.

I've added the `stop()` method in a separate branch:
https://github.com/wizardsardine/liana/compare/master...jp1ac4:liana:make-bitcoind-public

From some initial tests, it seems to work nicely and I'll commit those changes shortly.